docs/0.1-Glossary.md

```diff
@@ -0,0 +1,44 @@
+
+# 0.1 Glossary
+This glosary is intended to catalogue and define all terms used in our documentation which may not be evident to new users of the operator.
+
+By extension of this glossary, we may also use terms from the following glossaries:
+* [OLM](https://docs.openshift.com/container-platform/4.7/operators/understanding/olm-common-terms.html)
+* [Kubernetes](https://kubernetes.io/docs/reference/glossary/?fundamental=true)
+
+
+## Terms
+### Payer Account
+[AWS Organizations](https://aws.amazon.com/organizations/) give you a "unified bill" so to speak under a single account. Sub accounts are created under this "Payer Account", which is the highest-level account in this organization. SREs all should also have an IAM user with credentials on each payer account, which has AssumeRole Permissions as administrator into each of these sub-accounts.
```

Comment:
`SREs all should also have an IAM user with credentials on each payer account, which has AssumeRole Permissions as administrator into each of these sub-accounts.`

I'm not sure I'd refer to SREs here since this is a public-facing doc.